### PR TITLE
Consolidated tabbed content

### PR DESCRIPTION
**Pull request by cwarnermm**
Consolidated the two sections of tabbed content on this page into a single section of tabbed content for easier reading/scanning.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@justinegeffen - We'll want to apply tabbed content sparingly on our docs pages, and only when there's a clear use case for doing so. A couple of cases have been identified where tabbed content is offering the same content in multiple places on a docs page. We'll want to present that information once only to avoid confusion or ambiguity.

**Comment by justinegeffen** [comment]
@cwarnermm, good point and I agree.

Something I noted while reviewing https://github.com/mattermost/docs/pull/5248/files is that that PR mentions "From 6.1" and this PR's legacy wording is "in 6.0".

I propose having consistent wording in tabbed content to "From Mattermost version 6.0..." (instead of "In Mattermost version 6.0"). That isn't an exact mirror of the legacy content wording but I don't think they need to be similar in terms of phrasing. 

So we'll have 
"From Mattermost version 6.0..." 
"In Mattermost versions up to v5.39"

Thoughts? :)

**Comment by justinegeffen** [comment]
^^ I don't consider this a blocking comment for this consolidation so I am merging this. Let me know your thoughts and I can add the phrasing to the Handbook. :)

File: source/messaging/setting-your-status-availability.rst [hunk 1]
```diff
@@ -23,29 +23,23 @@ Let your team know whether you're available by setting a custom status and your
...
-  
 Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the channel sidebar and in conversations. 
 
 .. tabs::
 
   .. tab:: Mattermost v6.0 onwards
 
-     1. To set a custom status, select your Avatar in the top-right corner of the global header, then select **Set a Custom Status**.
-     2. Choose from a list of suggested statuses, reuse a recent status, or enter a new emoji and status, then select **Set Status**. The Speech bubble emoji 💬  is used by default if you don't specify an emoji. A custom status can be a maximum of 100 characters in length.
+      In Mattermost v6.0, set status and availability from your Profile in the top-right corner of the Global Header.
```
Comment (cwarnermm) [inline source/messaging/setting-your-status-availability.rst:32]:
```suggestion
      From Mattermost v6.0, set status and availability from your Profile in the top-right corner of the Global Header.
```

**Comment by cwarnermm** [comment]
I really like your proposal, @justinegeffen, thank you! I also agree with codifying it in the Handbook for future reference.

File: source/messaging/setting-your-status-availability.rst [hunk 2]
```diff
@@ -23,29 +23,23 @@ Let your team know whether you're available by setting a custom status and your
...
-
-  .. tab:: Mattermost v6.0 onwards
-
-      In Mattermost v6.0, set status and availability from your Avatar in the top-right corner of the Global Header.
-      
-  .. tab:: Mattermost v5.39 and earlier
-
-      In Mattermost versions up to 5.39, set status and availability from your avatar at the top of the channel sidebar.
-  
 Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the channel sidebar and in conversations.
```
Comment (justinegeffen) [inline source/messaging/setting-your-status-availability.rst:26]:
```suggestion
Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the Channel Sidebar and in conversations. 
```

File: source/messaging/setting-your-status-availability.rst [hunk 3]
```diff
@@ -23,29 +23,23 @@ Let your team know whether you're available by setting a custom status and your
...
-  
 Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the channel sidebar and in conversations. 
 
 .. tabs::
 
   .. tab:: Mattermost v6.0 onwards
 
-     1. To set a custom status, select your Avatar in the top-right corner of the global header, then select **Set a Custom Status**.
-     2. Choose from a list of suggested statuses, reuse a recent status, or enter a new emoji and status, then select **Set Status**. The Speech bubble emoji 💬  is used by default if you don't specify an emoji. A custom status can be a maximum of 100 characters in length.
+      From Mattermost v6.0, set status and availability from your Profile in the top-right corner of the Global Header.
```
Comment (justinegeffen) [inline source/messaging/setting-your-status-availability.rst:32]: Is it from your Profile or your Avatar?

File: source/messaging/setting-your-status-availability.rst [hunk 4]
```diff
@@ -23,29 +23,23 @@ Let your team know whether you're available by setting a custom status and your
...
+      
+      1. Select your Profile, then select **Set a Custom Status**.
```
Comment (justinegeffen) [inline source/messaging/setting-your-status-availability.rst:34]:
```suggestion
      1. Select your Avatar, then select **Set a Custom Status**.
```

**Review by justinegeffen — COMMENTED** [review]
One question but otherwise LGTM!

File: source/messaging/setting-your-status-availability.rst (hunk 3 above)
Comment (cwarnermm) [reply source/messaging/setting-your-status-availability.rst:32]: Great catch!

File: source/messaging/setting-your-status-availability.rst [hunk 5]
```diff
@@ -23,29 +23,23 @@ Let your team know whether you're available by setting a custom status and your
...
 -----------------------
 
+Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the Channel Sidebar and in conversations. 
+
 .. tabs::
 
   .. tab:: Mattermost v6.0 onwards
 
-      In Mattermost v6.0, set status and availability from your Avatar in the top-right corner of the Global Header.
+      From Mattermost v6.0, set status and availability from your Profile in the top-right corner of the Global Header.
```
Comment (cwarnermm) [inline source/messaging/setting-your-status-availability.rst:32]:
```suggestion
      From Mattermost v6.0, set status and availability from your Avatar in the top-right corner of the Global Header.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**